site.step-40-app-node.yml

```diff
@@ -190,6 +194,11 @@
     nginx_debug_request_header_name: "{{ mageops_debug_token_http_header }}"
     nginx_debug_request_query_param_name: "{{ mageops_debug_token_query_param }}"
     nginx_debug_request_cookie_name: "{{ mageops_debug_token_request_cookie }}"
+    nginx_qa_request_token: "{{ mageops_qa_token }}"
```

Comment:
Why do we need those `nginx_qa_request*` variables?
Looks like those are not used anywhere